bin/tictactoe

```diff
@@ -1,3 +1,53 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+# 0 player is computer vs computer
```

Comment:
For lines 4-53 we should separate this into a CLI class that lives in our lib folder. We would then be able to call this Class instance method on line 4 instead 

```
CLI.new.start_game
```